semantic_conventions.md

```diff
@@ -18,6 +18,7 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 |:--------------|:-----|:-------------------|
 | `component` | string  | The software package, framework, library, or module that generated the associated Span. E.g., `"grpc"`, `"django"`, `"JDBI"`. |
 | `db.instance` | string | Database instance name. E.g., In java, if the jdbc.url=`"jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `"customers"`. |
+| `db.collection` | string | Database collection name. E.g., for MongoDB this is the collection. For a SQL database, this is the name of the table. |
```

Comment:
> For a SQL database, this is the name of the table.

SQL query may join more than one table, how would a single tag work?

Reply:
Great point! Open to any suggestions around this! We can send the collection as a comma delimited set of strings perhaps? Alternatively, we can have this be applicable only to nosql style datastores which do not support joins. Thoughts?

Reply:
I guess my point is how much value such tag provides given the SQL join example. Also, instrumentation is often implemented as middleware that may only get access to the query string, and wouldn't know how to extract the table name. But in nosql drivers it may still be useful, eg in elastic api the caller specifies index names explicitly.